chaosk8s/pod/actions.py

```diff
@@ -79,7 +84,10 @@ def terminate_pods(label_selector: str = None, name_pattern: str = None,
     logger.debug("Picked pods '{p}' to be terminated".format(
         p=",".join([po.metadata.name for po in pods])))
 
-    body = client.V1DeleteOptions()
+    if grace_period >= 0:
```

Comment:
I wonder if we could collapse those conditionals into a single line? How does the API react to grace_period=-1 as a delete option? Or None?

Reply:
Not sure. I've just tried a `kubectl delete pod xxxx --grace-period=-1` and the operation works but I can't say what Kubernetes is doing (ignoring it vs using zero vs another) under the hood.

Reply:
LGTM. Python is not my bread and butter as you might have noticed. Any feedback is welcome.

Reply:
I actually, I thought your changes looked clean so I had assumed you coded in Python :)